modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -106,28 +114,293 @@ Execution types associated with the processed action
+| vm | listener
+.2+| websocket | inbound-listener | outbound-listener
+|===
+
+==== Mule Flow Generated by APIkit and Used for Routing APIkit Requests
+
+APIkit is a tool that simplifies the implementation of APIs by automatically generating a minimal set of Mule flows based on the API specification. Each APIkit router endpoint counts as a distinct Mule flow. These Mule flows do not have an event source and are used for the handling of HTTP requests for a particular API method and path.
+
+Here is an example of an APIkit request:
+
+This flow routes APIkit requests and handles the GET request in the `/reservation` path:
+
+[source,xml]
+----
+<flow name="get:\reservation:cocheras-puerto-madero-api-config">
+        <logger level="INFO" message="#[output json --- attributes.queryParams]" />
+</flow>
+----
+
+[[non-billable-flows]]
+==== Mule Flows That Don't Count Against Your Anypoint Platform Package Allotment 
+
+Mule flows that don't have an event source and aren't used for routing APIkit requests aren’t charged against your Anypoint Platform package allotment. These Mule flows are primarily used to modularize code.
+
+Example:
+
+[source,xml]
+----
+2.a - Flow with only a logger component
+<flow name="just-logging">
+        <logger level="INFO" message="#[output json --- attributes.queryParams]" />
+</flow>
+----
+
+[[mule-messages]]
+=== Total Mule Messages
+
+A Mule message is the data (the payload and its attributes) that passes through one or more Mule flows in an application. A Mule message is part of a Mule event, which is generated when the event source within a Mule flow is triggered. For example, a Mule event that consists of a Mule message is created when an HTTP listener receives a request or each time the scheduler component triggers an execution of the Mule flow. 
+Mule message processors in a Mule flow (such as core components, file read operations, or the HTTP request operations) can then retrieve, set, and process Mule message data that resides in the Mule event according to their configurations. 
+A Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a Mule message returns a new Mule message that consists of a message payload (the body of the message) and message attributes (metadata associated with the message).
+
+[[billable-mule-messages]]
+==== Mule Message Scenarios that Count Towards Your Anypoint Platform Package Allotment
+
+When an event source within a flow of a Mule application is triggered, the event source such as HTTP Listener or Scheduler, generates a Mule event that encapsulates a Mule message. The Mule message generated by the event source counts towards your Anypoint Platform package allotment. New instances of that message, which can be created during the processing of the original message as it moves through other processors in connected Mule flows, do not count towards your Anypoint Platform package allotment.
```

Comment:
```suggestion
When an event source within a flow of a Mule application is triggered, the event source such as HTTP listener or scheduler, generates a Mule event that encapsulates a Mule message. The Mule message generated by the event source counts toward your Anypoint Platform package allotment. New instances of that message, which can be created during the processing of the original message as it moves through other processors in connected Mule flows, don’t count towards your Anypoint Platform package allotment.
```

Reply:
I think these should be capped because they are referring to components.

Reply:
Then, do it the same on 295.